relayer/chain/ethereum/listener.go

```diff
@@ -157,7 +172,7 @@ func makeFilterQuery(contracts []Contract) geth.FilterQuery {
 
 	for _, contract := range contracts {
 		addresses = append(addresses, contract.Address)
-		signature := contract.ABI.Events["AppTransfer"].ID.Hex()
+		signature := contract.ABI.Events["AppTransfer"].Id().Hex()
```

Comment:
Does the `Id()` method actually exist? I don't see one defined in https://github.com/ethereum/go-ethereum/blob/master/accounts/abi/event.go.